phys2bids/tests/test_integration.py

```diff
@@ -197,11 +195,15 @@ def test_integration_heuristic(skip_integration, multifreq_lab_file):
     assert math.isclose(json_data['StartTime'], 3.6960,)
     assert json_data['Columns'] == ['time', 'Trigger', 'CO2']
 
+    shutil.copy(join(conversion_path, 'phys2bids_report.html'),
+                join(dirname(p2b.__file__), 'reporting', 'phys2bids_report.html'))
+    shutil.copy(join(conversion_path, 'phys2bids_report_log.html'),
+                join(dirname(p2b.__file__), 'reporting', 'phys2bids_report_log.html'))
+
     # Remove generated files
     shutil.rmtree(test_path_output)
     shutil.rmtree(conversion_path)
-    for filename in glob.glob(join(test_path, 'Test1_multifreq_onescan*')):
-        remove(filename)
+    shutil.rmtree(test_path)
```

Comment:
I think here we the problem,  you are deleting the testpath in the first integration test, and therefore there is an error later when trying to download to that directory